packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -2342,6 +2375,9 @@ Future<TimeOfDay?> showTimePicker({
   String? cancelText,
   String? confirmText,
   String? helpText,
+  String? errorInvalidText,
```

Comment:
Can you add some documentation for these parameters above? I'm thinking line 2322 where this currently exists: 
```dart
/// Optional strings for the [helpText], [cancelText], and [confirmText] can be
/// provided to override the default values.
```

Reply:
Done